ssl/ssl_key_share.cc

```diff
@@ -303,12 +531,21 @@ CONSTEXPR_ARRAY NamedGroup kNamedGroups[] = {
     {NID_CECPQ2, SSL_CURVE_CECPQ2, "CECPQ2", "CECPQ2"},
 };
 
+CONSTEXPR_ARRAY uint16_t kPQGroups[] = {
+    SSL_CURVE_CECPQ2,
+    SSL_CURVE_KYBER512_R3,
```

Comment:
should we add Kyber to the named groups as well?

Reply:
Adding it to the named groups will make it immediately publicly available for use in handshakes. I'm assuming we wouldn't want that because (a) we do not (yet) want kyber as a standalone group to be used, and (b) other testing (notably the Go integration tests) would need to be added.